CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 3.2.0 (2018-07-03)
+
+This release introduces automatic tracking of sessions, which by
+default are captured for each HTTP request received via the Servlet API
+To disable this data collection, call `bugsnag.setAutoCaptureSessions(false)`.
```

Comment:
this line break needs a double break if you want a new line in the rendered markdown, or some punctuation